packages/create-sitecore-jss/src/templates/nextjs/scripts/generate-component-builder/plugins/feaas.ts

```diff
@@ -0,0 +1,28 @@
+import { ComponentBuilderPlugin, ComponentBuilderPluginConfig } from '..';
+
+/**
+ * Provides Sitecore Components (FEaaS) packages configuration
+ */
+class FEaaSPlugin implements ComponentBuilderPlugin {
+  order = 1;
+
+  exec(config: ComponentBuilderPluginConfig) {
+    config.packages.push({
+      name: '@sitecore-jss/sitecore-jss-nextjs',
+      components: [
+        {
+          componentName: 'BYOCWrapper',
```

Comment:
Should this plugin be a part of sxa addon instead? Both FEAAS and BYOC wrappers are only added by SXA on Sitecore side.

Reply:
It's a good question @art-alexeyenko. Currently, only SXA-based "Wrapper" components are supported in Pages, but in the future (if/when FEAAS is fully supported for non-SXA sites), we would likely have a `feaas.ts` in both base `nextjs` (registers `BYOCComponent` and `FEaaSComponent`) and `nextjs-sxa` (registers `BYOCWrapper` and `FEaaSWrapper`) templates.

However, probably a change we can make then? For now, I don't mind the feaas plugin in base.